lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -107,4 +107,14 @@ around 'default_priority' => sub {
     return $self->get_column('default_priority') // ($self->parent ? $self->parent->default_priority : OpenQA::Schema::JobGroupDefaults::PRIORITY);
 };
 
+sub rendered_description {
+    my ($self) = @_;
+
+    if ($self->description) {
+        my $m = CommentsMarkdownParser->new;
+        return Mojo::ByteStream->new($m->markdown($self->description));
+    }
+    return 0;
```

Comment:
the problem is that a perl function without explicit return returns the last statement's return. And you will want to avoid debugging problems that are raised because you added a debug statement in this function.

other languages have different defaults for returns.